presto-main/src/main/java/com/facebook/presto/spiller/SpillerFactoryWithStats.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spiller;
+
+import org.weakref.jmx.Managed;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public abstract class SpillerFactoryWithStats
+        implements SpillerFactory
+{
+    protected final AtomicLong spilledBytes = new AtomicLong();
+
+    @Override
+    @Managed
+    public long getSpilledBytes()
```

Comment:
We thought about it and probably at some point we will add this (we have even ticket for that). But baby steps first :-)